src/packages/documents/document-types/workspace/views/design/document-type-workspace-view-edit-property.element.ts

```diff
@@ -40,11 +49,36 @@ export class UmbDocumentTypeWorkspacePropertyElement extends UmbLitElement {
 	@property({ type: Boolean })
 	public inherited?: boolean;
 
+	#dataTypeRepository = new UmbDataTypeRepository(this);
+
 	#modalRegistration;
+	private _modalManagerContext?: typeof UMB_MODAL_MANAGER_CONTEXT_TOKEN.TYPE;
 
 	@state()
 	protected _modalRoute?: string;
 
+	@state()
+	protected _editDocumentTypePath?: string;
+
+	@property()
+	public get modalRoute() {
+		return this._modalRoute;
+	}
+
+	@property({ type: String, attribute: 'data-owner-document-type-id' })
```

Comment:
I would say we can remove the word 'data' from these two. Makes it easier to read.